jpa/eclipselink.jpa.test.jse/src/it/java/org/eclipse/persistence/jpa/test/sharedcache/TestSharedCacheWithNonCacheable.java

```diff
@@ -0,0 +1,186 @@
+/********************************************************************************
+ * Copyright (c) 2020 IBM Corporation. All rights reserved.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License v. 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0,
+ * or the Eclipse Distribution License v. 1.0 which is available at
+ * http://www.eclipse.org/org/documents/edl-v10.php.
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR BSD-3-Clause
+ ******************************************************************************/
+
```

Comment:
can you follow the convention used in other files wrt formatting of license header? I do not think this is going to pass the copyright check - verify by running `mvn -Pcoding-standards glassfish-copyright-mave-plugin:check` or `mvn -Pcoding-standards glassfish-copyright-mave-plugin:copyright` (the latter will show wrong files but won't fail the build in case of errors) - note that the check may take up to 2hours to finish

Reply:
Hello @lukasj, thank you for your attention, I fixed the copyright as you suggested.
I ran:
`mvn -X -Pcoding-standards glassfish-copyright:copyright -pl foundation/org.eclipse.persistence.core`
And can confirm it ends with a `BUILD SUCCESS` and no reported errors.

I rebased master on my branch in order to be up to date

Reply:
Updated PR description. Everything was already in bugzilla + test case but it's better to sum up here for future reference